src/network/serverpackethandler.cpp

```diff
@@ -1358,10 +1395,13 @@ void Server::handleCommand_Interact(NetworkPacket* pkt)
 		Check that target is reasonably close
 		(only when digging or placing things)
 	*/
-	if (action == 0 || action == 2 || action == 3) {
-		float d = player_pos.getDistanceFrom(pointed_pos_under);
-		float max_d = BS * 14; // Just some large enough value
-		if (d > max_d) {
+	float d = player_pos.getDistanceFrom(pointed_pos_under);
```

Comment:
This whole block has nothing to do with the actual change, why edit it?

In fact , the new way is worse style because it needlessly creates a variable that's never read, and puts three variables into a too large scope.